keystorage/aws/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+apply plugin: 'java-library'
+apply plugin: 'java-test-fixtures'
+
+jar {
+  archiveBaseName = calculateJarName(project)
+  manifest {
+    attributes(
+      'Specification-Title': archiveBaseName.get(),
+      'Specification-Version': rootProject.version,
+      'Implementation-Title': archiveBaseName.get(),
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+testFixturesJar {
+  archiveBaseName = calculateJarName(project)
+}
+
+dependencies {
+
+  implementation platform('software.amazon.awssdk:bom:2.17.125')
+
+  implementation 'io.vertx:vertx-core'
```

Comment:
Looks like this only needed for parsing the json object? If that's the case it would be better to just use Jackson which Vertx is using under the hood anyway.